frontend/coprs_frontend/coprs/views/coprs_ns/coprs_builds.py

```diff
@@ -64,7 +64,10 @@ def render_copr_build(build_id, copr):
 @req_with_copr
 @req_with_pagination
 def copr_builds(copr, page=1):
-    _preload = copr.group, copr.forked_from, copr.dirs
+    _preload = copr.group, copr.dirs
+    _preload = copr.forked_from and copr.forked_from.group
+    _preload = copr.forked_from and copr.forked_from.user
```

Comment:
I'm not sure here, but overriding the `_preload` variable might negatively affect the objects that it referred to before

Reply:
I doubt it. IMHO the important part is accessing those relationships and therefore forcing the query. I didn't try, but I'd expect this to work even if we didn't store the values to `_preload`